templates/zerver/help/view-messages-containing-files-or-links.md

```diff
@@ -0,0 +1,10 @@
+# View messages containing files or links
+
+By using Zulip's [search feature](/help/search-for-messages), you can find all
+messages in your organization that contain any files or links.
```

Comment:
`has:attachment` and `has:link` are mutually exclusive, I think. They won't show the same messages unless you combine them.